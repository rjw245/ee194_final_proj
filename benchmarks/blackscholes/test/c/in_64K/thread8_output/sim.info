{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread8_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-pthreads/bin/blackscholes',
             '8',
             '../../../inputs/in_64K.txt',
             'thread8_output/output.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (12451.977011,
            129.51831,
            136628,
            0,
            0,
            0,
            224559,
            8,
            0,
            7896,
            15192,
            0,
            0,
            0,
            19002768,
            5677417),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/c/in_64K/thread8_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-pthreads/bin/blackscholes 8 ../../../inputs/in_64K.txt thread8_output/output.txt'],
 't_elapsed': 8150.336498975754,
 't_start': 1461613856.412385,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 935202816L}
