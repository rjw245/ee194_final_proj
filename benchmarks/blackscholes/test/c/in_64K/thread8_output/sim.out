                                     | Core 0      | Core 1     | Core 2     | Core 3     | Core 4     | Core 5     | Core 6     | Core 7    
  Instructions                       |  1338782223 |  593153606 |  593168782 |  593147206 |  593184460 |  593171060 |  593164760 |  593157749
  Cycles                             |   872020884 |  872020884 |  872020884 |  872020884 |  872020884 |  872020884 |  872020884 |  872020884
  IPC                                |        1.54 |       0.68 |       0.68 |       0.68 |       0.68 |       0.68 |       0.68 |       0.68
  Time (ns)                          |   327827400 |  327827400 |  327827400 |  327827400 |  327827400 |  327827400 |  327827400 |  327827400
  Idle time (ns)                     |     2995704 |  175796688 |  176195578 |  175962031 |  175720355 |  172403329 |  175398671 |  175230469
  Idle time (%)                      |        0.9% |      53.6% |      53.7% |      53.7% |      53.6% |      52.6% |      53.5% |      53.5%
Branch predictor stats               |             |            |            |            |            |            |            |           
  num correct                        |   152851133 |   34929042 |   34946526 |   34941034 |   34917642 |   34874368 |   34894463 |   34892693
  num incorrect                      |     8032755 |     979037 |     961723 |     967145 |     989941 |    1033615 |    1013220 |    1015329
  misprediction rate                 |       4.99% |      2.73% |      2.68% |      2.69% |      2.76% |      2.88% |      2.82% |      2.83%
  mpki                               |        6.00 |       1.65 |       1.62 |       1.63 |       1.67 |       1.74 |       1.71 |       1.71
TLB Summary                          |             |            |            |            |            |            |            |           
  I-TLB                              |             |            |            |            |            |            |            |           
    num accesses                     |   188985621 |   66976174 |   66978564 |   66975974 |   66980307 |   66978107 |   66977507 |   66977679
    num misses                       |         117 |         23 |         23 |         23 |         23 |         23 |         23 |         23
    miss rate                        |       0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |        0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  D-TLB                              |             |            |            |            |            |            |            |           
    num accesses                     |   445703800 |  190177181 |  190180094 |  190175781 |  190182971 |  190180171 |  190179271 |  190177050
    num misses                       |     2863599 |    3772864 |    2179184 |    3674064 |    2176766 |    3824365 |    2176966 |    3674852
    miss rate                        |       0.64% |      1.98% |      1.15% |      1.93% |      1.14% |      2.01% |      1.14% |      1.93%
    mpki                             |        2.14 |       6.36 |       3.67 |       6.19 |       3.67 |       6.45 |       3.67 |       6.20
  L2 TLB                             |             |            |            |            |            |            |            |           
    num accesses                     |     2863716 |    3772887 |    2179207 |    3674087 |    2176789 |    3824388 |    2176989 |    3674875
    num misses                       |        7673 |        827 |        731 |       9823 |        723 |     103322 |        723 |        713
    miss rate                        |       0.27% |      0.02% |      0.03% |      0.27% |      0.03% |      2.70% |      0.03% |      0.02%
    mpki                             |        0.01 |       0.00 |       0.00 |       0.02 |       0.00 |       0.17 |       0.00 |       0.00
Cache Summary                        |             |            |            |            |            |            |            |           
  Cache L1-I                         |             |            |            |            |            |            |            |           
    num cache accesses               |   188985621 |   66976174 |   66978564 |   66975974 |   66980307 |   66978107 |   66977507 |   66977679
    num cache misses                 |      252605 |       1732 |       1762 |       1732 |       1732 |       1732 |       1732 |       1733
    miss rate                        |       0.13% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |        0.19 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  Cache L1-D                         |             |            |            |            |            |            |            |           
    num cache accesses               |   445835067 |  190177187 |  190180102 |  190175787 |  190182976 |  190180176 |  190179276 |  190177055
    num cache misses                 |      748900 |     549821 |     565716 |     552333 |     549625 |     544984 |     545115 |     565379
    miss rate                        |       0.17% |      0.29% |      0.30% |      0.29% |      0.29% |      0.29% |      0.29% |      0.30%
    mpki                             |        0.56 |       0.93 |       0.95 |       0.93 |       0.93 |       0.92 |       0.92 |       0.95
  Cache L2                           |             |            |            |            |            |            |            |           
    num cache accesses               |     1001576 |     551557 |     567480 |     554065 |     551363 |     546721 |     546851 |     567117
    num cache misses                 |      135091 |      17772 |      17778 |      22237 |      18062 |      18361 |      17868 |      22096
    miss rate                        |      13.49% |      3.22% |      3.13% |      4.01% |      3.28% |      3.36% |      3.27% |      3.90%
    mpki                             |        0.10 |       0.03 |       0.03 |       0.04 |       0.03 |       0.03 |       0.03 |       0.04
  Cache L3                           |             |            |            |            |            |            |            |           
    num cache accesses               |      135171 |      17777 |      17780 |      22237 |      18068 |      18366 |      17872 |      22101
    num cache misses                 |       70703 |        637 |        653 |        697 |       3943 |       3729 |       3720 |       3838
    miss rate                        |      52.31% |      3.58% |      3.67% |      3.13% |     21.82% |     20.30% |     20.81% |     17.37%
    mpki                             |        0.05 |       0.00 |       0.00 |       0.00 |       0.01 |       0.01 |       0.01 |       0.01
DRAM summary                         |             |            |            |            |            |            |            |           
  num dram accesses                  |       36267 |          0 |          0 |          0 |      36269 |          0 |          0 |          0
  average dram access latency (ns)   |       61.02 |        inf |        inf |        inf |      60.88 |        inf |        inf |        inf
  average dram queueing delay        |        7.60 |       0.00 |       0.00 |       0.00 |       7.45 |       0.00 |       0.00 |       0.00
  average dram bandwidth utilization |       0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
Coherency Traffic                    |             |            |            |            |            |            |            |           
  num loads from dram                |        8574 |        556 |        547 |        533 |         49 |         39 |         46 |         26
  num loads from dram cache          |           0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from remote cache        |        2081 |         29 |         35 |         32 |       3185 |       3121 |       3128 |       3163
